src/pretix/api/serializers/voucher.py

```diff
@@ -64,14 +66,15 @@ def to_representation(self, val: Seat):
 
 class VoucherSerializer(I18nAwareModelSerializer):
     seat = SeatGuidField(allow_null=True, required=False)
+    budget_used = serializers.DecimalField(read_only=True, max_digits=13, decimal_places=2, min_value=Decimal('0.00'))
```

Comment:
Do we need to add this for `budget` as well?

```suggestion
    budget = serializers.DecimalField(max_digits=13, decimal_places=2, min_value=Decimal('0.00'))
    budget_used = serializers.DecimalField(read_only=True, max_digits=13, decimal_places=2, min_value=Decimal('0.00'))
```

Reply:
At least for me, budget is already correcting resolving to a Decimal-String in the API-view...